Sources/RxMoya/RxMoyaProvider.swift

```diff
@@ -69,3 +74,10 @@ public extension RxMoyaProvider {
         }
     }
 }
+
+public extension MoyaProvider {
+    
+    public var rx: RxMoyaProvider<Target> {
```

Comment:
In Rx to make something in the `rx` prefix, one would usually do:
```swift
extension Reactive where Base: MoyaProvider {
    func request() -> Observable<Response> {...}
}
```

It is probably similar in ReactiveSwift.